lib/ar_state_machine.rb

```diff
@@ -293,14 +293,14 @@ def append_transitions_callback(states, method_or_block, rollback_on_failure, me
       end
     end
 
-    def before_transition_to(to, method = nil, rollback_on_failure = true, &block)
+    def before_transition_to(to, method = nil, rollback_on_failure: true, &block)
```

Comment:
not that we have any test on it rn. but probs should have a test if it easy enough?